### PR TITLE
Switch CI to master branch of the examples

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/examples/issues/285

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,12 @@ dist: xenial
 language: rust
 matrix:
   include:
+    - os: linux
```
Comment (GuillaumeGomez) [inline .travis.yml]: In this change really necessary? If not, it'll just make more diff with the master branch...

Comment (EPashkin) [reply .travis.yml]: Not necessary, I will revert it

**Comment by EPashkin** [comment]
CI passed

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**